.circleci/Dockerfile.debian

```diff
@@ -18,8 +18,7 @@ RUN apt-get --quiet update && \
         libffi-dev \
         libssl-dev \
         libyaml-dev \
-        virtualenv \
-        tor
```

Comment:
Is this a spurious change?

(Or: why does the debian docker image need to exclude Tor for anything "Nix" to work?)

Reply:
Ah, good catch.  Indeed, this is not required for the new Nix-based integration tests to work.  Instead, I think it is a left-over from when this branch (or maybe an earlier one that I copied some changes from) also _removed_ the Debian/Docker-based integration test job from CI.

So I could either revert this Tor removal _or_ expand this branch to also remove the Debian/Docker integration tests CI job.

And the "circleci: integration" job should really be red here but it isn't yet because the Docker image hasn't been manually re-generated to reflect this Dockerfile change :/

Any preference on which way to take this @meejah ?

Reply:
Maybe both?

Revert it here, but a new PR to expand Nix stuff to run Tor too would be interesting (I think that's approximately what you mean?)